utils/order-utils/src/constants.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
   SECONDS_IN_DAY: 86400,
   EMPTY_ADDRESS: '0x0000000000000000000000000000000000000000',
   HEAD: '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF',
```

Comment:
why are some of the f's lower case and some upper case?

Reply:
Its a check summed Ethereum address